packages/core/src/schema/models/workspaces.ts

```diff
@@ -12,3 +13,26 @@ export const workspaces = latitudeSchema.table('workspaces', {
   }),
   ...timestamps(),
 })
+
+export const plansEnum = latitudeSchema.enum('subscription_plans', [
+  SubscriptionPlan.HobbyV1,
+  SubscriptionPlan.HobbyV2,
```

Comment:
Why v1 and v2? Why not different names?

Reply:
Not really important tbh but at least an incremental pattern gives some extra info